src/backend/distributed/worker/worker_drop_protocol.c

```diff
@@ -114,7 +114,7 @@ worker_drop_distributed_table(PG_FUNCTION_ARGS)
 		add_exact_object_address(&foreignServerObject, objects);
 
 		/* drop both the table and the server */
-		performMultipleDeletions(objects, DROP_RESTRICT,
+		performMultipleDeletions(objects, DROP_CASCADE,
```

Comment:
We already drop the server and the table by adding two objects. Why do we need to change `DROP_CASCADE`?